.github/workflows/CI.yml

```diff
@@ -52,6 +52,10 @@ jobs:
             arch: x86
           - provider: 'mkl'
             threads: '2'
+          # x86 has issues with MKL 2025
+          - os: windows-latest
+            provider: 'mkl'
```

Comment:
```suggestion
          - provider: 'mkl'
```

Reply:
And delete https://github.com/JuliaMath/FFTW.jl/blob/7e845933e80e70b6a06576e7abcc0206c9ff51f7/.github/workflows/CI.yml#L49-L54